### PR TITLE
Add Lisa's entry for 21.01.2022

### PR DESCRIPTION
**Pull request by glsignal**

### CODE REVIEW

File: blog/lisa-event-marketing-manager.html [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+<html data-wf-page="5f5f7eccd3d6ead7d8f2d868" data-wf-site="5f5f7eccd3d6ea00abf2d859" lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>The people behind Alasco: Meet our Frontrunners!</title>
+    <meta content="Sophia as a UI/UX Designer" name="description">
+    <meta content="How to become a frontrunner!" property="og:title">
+    <meta content="Sophia as a UI/UX Designer" property="og:description">
+    <meta content="How to become a frontrunner! - Alasco Inside" property="twitter:title">
+    <meta content="Sophia as a UI/UX Designer" property="twitter:description">
```
Comment (sixfwa) [inline blog/lisa-event-marketing-manager.html]: Would change the meta tags to fit with the blog ;)

**Review by sixfwa — CHANGES_REQUESTED** [review]
Thank you so much for add this :). Ive just added one change in regarding the files. However there is one more in the `index.html`. There is an area with the heading titled "Latest Blogposts". Could you please add in this blog post there :) 

Thanks again

File: blog/lisa-event-marketing-manager.html (hunk 1 above)
Comment (glsignal) [reply blog/lisa-event-marketing-manager.html]: Thanks good catch! Updated the notes too, and also added one for the breadcrumb link

Comment (sixfwa) [reply blog/lisa-event-marketing-manager.html]: awesome thank you :)

**Review by sixfwa — APPROVED** [review]
🚀 🚀 let's get this literature deployed

**Comment by glsignal** [comment]
Prose: released

**Pull request merged**